dashboard/app/views/shared/_user_stats.html.haml

```diff
@@ -1,7 +1,6 @@
 - script = @script || (user && user.primary_script) || Script.twenty_hour_script
 - script_summary = script.summarize
-- async = local_assigns[:async]
-- progress = summarize_user_progress(script, user, async) || {}
+- progress = summarize_user_progress(script, user) || {}
```

Comment:
An "async" `_user_stats.html.haml` render was one that used HAML to render the markup, then went back and filled in progress via jQuery.  Now that the React component is used everywhere we have all the state tracked in one place 😄.

I can't remove this partial yet because it's still used for the teacher dashboard student progress view.  In that scenario the page is never cached, so we never need to populate progress async.